core/templates/components/ck-editor-helpers/customize-rte-component-modal.template.html

```diff
@@ -15,6 +15,9 @@ <h3>Customize This Component</h3>
         </div>
       </div>
     </form>
+    <div ng-if="disableSaveButtonForLinkRte()" class="link-error-message">
+      It seems like clicking on this link will lead the user to a different URL than the text specifies. Please change the text.
```

Comment:
Can we move this text to a constant?

Reply:
Do you mean an `I18N` constant? `rte-helper.service.ts` hasn't been upgraded to Angular yet so I'm not sure how to import `ngx-translate`.